style.css

```diff
@@ -0,0 +1,1427 @@
+  height: 250px;
+  color: #fff;
+  position: relative;
+}
+
+.apps h2,
+.tech h2,
+.gear h2,
+.creative h2,
+.podium h2,
+.insights h2,
+.launch h2,
+.hard-fork h2,
+.distract h2 {
+  font-weight: 900;
+  color: #f42;
+}
+
+.apps-p1-text,
+.tech-p1-text,
+.gear-p1-text,
+.creative-p1-text,
+.podium-p1-text,
+.insights-p1-text,
+.launch-p1-text,
+.hard-fork-p1-text,
+.distract-p1-text {
+  bottom: 0;
+  left: 0.8rem;
+  position: absolute;
+}
+
+.apps-p2-text p,
+.tech-p2-text p,
+.gear-p2-text p,
+.creative-p2-text p,
+.podium-p2-text p,
+.insights-p2-text p,
+.launch-p2-text p,
+.hard-fork-p2-text p,
+.distract-p2-text p {
+  font-size: 14px;
+  margin-bottom: 0;
+  font-weight: 700;
+}
+
+.latest-deals p {
+  font-weight: 700;
+  color: rgb(99, 99, 99);
+  margin-top: 0;
+  font-size: 100%;
+}
+
+.last-line p {
+  padding-top: 60px;
+  color: white;
+}
+
+.apps-p1-text h3 h5 p,
+.tech-p1-text h3 h5 p,
+.gear-p1-text h3 h5 p,
+.creative-p1-text h3 h5 p,
+.podium-p1-text h3 h5 p,
+.insights-p1-text h3 h5 p,
+.launch-p1-text h3 h5 p,
+.hard-fork-p1-text h3 h5 p,
+.distract-p1-text h3 h5 p {
+  margin-block-end: 0.3rem;
+  margin-block-start: 0.3rem;
+}
+
+.apps-p2,
+.tech-p2,
+.gear-p2,
+.creative-p2,
+.podium-p2,
+.insights-p2,
+.launch-p2,
+.hard-fork-p2,
+.distract-p2 {
+  background-color: rgba(255, 68, 34, 0.06);
+  margin-top: 2px;
+  justify-content: space-between;
+  display: flex;
+}
+
+.apps-p2-text,
+.tech-p2-text,
+.gear-p2-text,
+.creative-p2-text,
+.podium-p2-text,
+.insights-p2-text,
+.launch-p2-text,
+.hard-fork-p2-text,
+.distract-p2-text {
+  padding: 0 5px 0 5px;
+  display: flex;
+  flex-direction: column;
+  color: #f42;
+}
+
+.apps-p2-text h5,
+.tech-p2-text h5,
+.gear-p2-text h5,
+.creative-p2-text h5,
+.podium-p2-text h5,
+.insights-p2-text h5,
+.launch-p2-text h5,
+.hard-fork-p2-text h5,
+.distract-p2-text h5 {
+  font-weight: 200;
+  margin-bottom: 0;
+}
+
+.hard-fork ul,
+.gear ul,
+.tech ul,
+.creative ul,
+.podium ul,
+.insights ul,
+.launch ul,
+.apps ul,
+.distract ul {
+  list-style: none;
+  padding-left: 1em;
+}
+
+.apps ul li,
+.gear ul li,
+.tech ul li,
+.creative ul li,
+.podium ul li,
+.insights ul li,
+.launch ul li,
+.hard-fork ul li,
+.distract ul li {
+  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 1.5;
+  padding: 2px 0 4px 0;
+  color: #555;
+}
+
+.hard-fork ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.apps ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.gear ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
+
+.tech ul li::before {
+  color: #f42;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+  content: "\2022";
+}
```

Comment:
## Aim to Stay `DRY`
- [ ] The lines of code from lines `#601` to `#673` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant? A good suggestion would be to have a new `class` for the repeating rule.